OpenRA.Mods.Common/Traits/Render/WithSpriteTurret.cs

```diff
@@ -139,11 +148,16 @@ void ITick.Tick(Actor self)
 			Tick(self);
 		}
 
-		public void PlayCustomAnimation(Actor self, string name, Action after = null)
+		public void PlayCustomAnimation(Actor self, string name, Action after = null, bool isUninterruptible = false)
 		{
+			if (UninterruptibleAnimationPlaying)
+				return;
```

Comment:
I expect this to break any code that relies on the `after` action running.

Reply:
Would it be enough to remove this here and let the places that want to start some other anim check this instead?